Libraries/process/Sources/DefaultLauncher.cpp

```diff
@@ -13,6 +13,9 @@
 #include <sys/wait.h>
 #include <unistd.h>
 
+// In most cases, size of pipe will be greater than one page,
+#define kPIPE_BUFFER_SIZE 4096
```

Comment:
The lowercase `k` seems weird for a constant like this, but if that's the style used in the project it's fine :/